pkg/cephmgr/cephd/cephd.go

```diff
@@ -12,7 +12,33 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
+A couple of functions below (rados_connect and rados_mon_command) came from
```

Comment:
the scope of the functions from go-ceph is larger than the TODO comment indicated.  that initial TODO to track go-ceph licensing went stale as more were added in.  if a general attribution to go-ceph could be indicated, instead of specific function names, that would be best.